src/module/syscalls.h

```diff
@@ -28,7 +28,45 @@
 
 #define KRF_DEFINE(sys) asmlinkage krf_sys_##sys
 
-#define KRF_TARGETED() ((current->personality & krf_personality) != 0)
+#define KRF_TARGETED()                                                                             \
```

Comment:
Let's go ahead and make this an inline function, below the `extern`s:

```c
static __always_inline int KRF_TARGETED(void) {
  // ...
}
```

Might also be worth updating `module/codegen/codegen` so that we can make it `krf_targeted` instead of all caps.